iroh-resolver/src/chunker.rs

```diff
@@ -35,6 +35,8 @@ pub enum ChunkerStream<'a> {
     Rabin(LocalBoxStream<'a, io::Result<Bytes>>),
 }
 
+unsafe impl<'a> Send for ChunkerStream<'a> {}
```

Comment:
No, this is not safe to do. The streams need to change to `BoxStream` to make this work